### PR TITLE
Feature/abd edit profile view

### LINKED ISSUE #39: Add form view for updating profile information
When a user clicks "Edit profile info", they are taken to a new view that includes a form for updating profile information 

- [x] Connect "Edit profile info" to route to that new page
- [x] Create the page (i.e. the form)

### PR DESCRIPTION
**Pull request by Abdeboskey**
# Crate Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

* Create the display to edit the profile. 
* Create  file `EditProfile.js` for new form component
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [x] No
- [ ] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:

* These changes were all tested with devTools in the browser.

# Message for reviewer:

### Please see note about `package-lock.json` above ☝️

# Checklist:
- [ ] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

css be like

### CODE REVIEW

File: code/web/src/modules/user/EditProfile.js [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+  //         isLoading: false,
+  //       })
+
+  //       window.setTimeout(() => {
+  //         this.props.messageHide()
+  //       }, 5000)
+  //     })
+  // }
+
+  render() {
```
Comment (rwilliams659) [inline code/web/src/modules/user/EditProfile.js:135]: UI looks awesome, glad we got this done!

File: code/web/src/modules/user/helperData/statesList.js [hunk 2]
```diff
@@ -0,0 +1,206 @@
+export const statesList = [
```
Comment (rwilliams659) [inline code/web/src/modules/user/helperData/statesList.js:1]: Great call to make this a separate file

**Review by rwilliams659 — APPROVED** [review]
Great work team!

File: code/web/src/modules/user/EditProfile.js [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+          <GridCell style={{ padding: '2em', textAlign: 'center', maxWidth: '20vw', marginRight: '24%' }}>
+            <H3 font='secondary'>Edit profile</H3>
+          </GridCell>
+          
+          <GridCell style={{ padding: '2em', textAlign: 'center', maxWidth: '9vw' }}>
+            <Button theme='secondary' onClick={this.props.logout} style={{ marginLeft: '1em' }}>Logout</Button>
+          </GridCell>
+        </Grid>
+        
+        <form style={{
```
Comment (JoshSevy) [inline code/web/src/modules/user/EditProfile.js:154]: Great Job Team! Thanks for pulling the weight during my technical interview. You all are the best!

File: code/web/src/modules/user/helperData/statesList.js [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+  {
+    name: 'Arkansas',
+    code: 'AR'
+  },
+  {
+    name: 'California',
+    code: 'CA'
+  },
+  {
+    name: 'Colorado',
```
Comment (JoshSevy) [inline code/web/src/modules/user/helperData/statesList.js:23]: Love the dropdown. Great work!

**Review by JoshSevy — APPROVED** [review]
Thanks again. Great work, both of you.

File: code/web/src/modules/user/EditProfile.js [hunk 5]
```diff
@@ -0,0 +1,313 @@
...
+}
+
+function editProfileState(state) {
+  return {
+    user: state.user,
+  };
+}
+
+export default withRouter(connect(editProfileState, { logout, messageShow, messageHide, upload })(EditProfile))
+
```
Comment (sienna-kopf) [inline code/web/src/modules/user/EditProfile.js:312]: I would love a walkthrough of this file at some point!! Looks great from what I understand-- but I think an explanation would enhance my learning significantly

File: code/web/src/modules/user/helperData/statesList.js (hunk 2 above)
Comment (sienna-kopf) [inline code/web/src/modules/user/helperData/statesList.js:1]: omg you have to code this all out 😣😣😣
 Nice work looks great!

File: code/web/src/modules/user/EditProfile.js [hunk 6]
```diff
@@ -0,0 +1,313 @@
...
+import { primary } from '../../ui/common/fonts'
+
+// App Imports
+// import admin from '../../../setup/routes/admin'
+import { routeImage } from '../../setup/routes'
+import { renderIf, slug } from '../../setup/helpers'
+import { logout } from './api/actions'
+import { upload, messageShow, messageHide } from '../common/api/actions'
+import { APP_URL } from '../../setup/config/env'
+import { statesList } from './helperData/statesList'
```
Comment (sienna-kopf) [inline code/web/src/modules/user/EditProfile.js:25]: Really nice job figuring out what you had to import to make this work. I can imagine that being a very tricky task.

File: code/web/package.json [hunk 7]
```diff
@@ -41,15 +41,19 @@
     "dotenv": "^8.2.0",
     "express": "5.0.0-alpha.7",
     "gql-query-builder": "3.3.0",
+    "i": "^0.3.6",
```
Comment (Strompy) [inline code/web/package.json:44]: Curious what "i" is?

File: code/web/src/modules/user/EditProfile.js (hunk 6 above)
Comment (sienna-kopf) [reply code/web/src/modules/user/EditProfile.js:25]: Final comment

**Review by sienna-kopf — APPROVED** [review]

File: code/web/src/modules/user/EditProfile.js [hunk 8]
```diff
@@ -0,0 +1,313 @@
...
+  //     .then(() => {
+  //       this.setState({
+  //         isLoading: false,
+  //       })
+
+  //       window.setTimeout(() => {
+  //         this.props.messageHide()
+  //       }, 5000)
+  //     })
+  // }
```
Comment (Strompy) [inline code/web/src/modules/user/EditProfile.js:133]: Is this comment a work in progress or something that may not be needed?

File: code/web/src/modules/user/EditProfile.js [hunk 9]
```diff
@@ -0,0 +1,313 @@
...
+                <label htmlFor='file'>
+                  Upload profile picture ⤵ {' '}
+                  <input
+                    type='file'
+                    onChange={this.onUpload}
+                    style={{ marginTop: '1em'}}
+                  />
+                </label>
+              </div>
+
```
Comment (Strompy) [inline code/web/src/modules/user/EditProfile.js:215]: The upload file is so cool!

File: code/web/src/modules/user/helperData/statesList.js [hunk 10]
```diff
@@ -0,0 +1,206 @@
...
+    code: 'WV'
+  },
+  {
+    name: 'Wisconsin',
+    code: 'WI'
+  },
+  {
+    name: 'Wyoming',
+    code: 'WY'
+  }
```
Comment (Strompy) [inline code/web/src/modules/user/helperData/statesList.js:205]: Dang that's a lot of typing state names!

**Review by Strompy — APPROVED** [review]

**Pull request merged**